src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,29 @@
 import React from 'react';
 import './MoviesList.scss';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard';
+import { movieType } from '../types';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard
+        title={movie.title}
+        description={movie.description}
+        imgUrl={movie.imgUrl}
+        imdbUrl={movie.imdbUrl}
+        imdbId={movie.imdbId}
+      />
```

Comment:
use spread operator
https://medium.com/@kevinyckim33/jsx-spread-operator-component-props-meaning-3c9bcadd2493

```suggestion
      <MovieCard {...movie} />
```

Reply:
also add key
https://reactjs.org/docs/lists-and-keys.html